packages/dds/tree/src/simple-tree/leafNodeSchema.ts

```diff
@@ -77,9 +77,9 @@ export interface LeafSchema<Name extends string, T extends TreeLeafValue>
 	extends TreeNodeSchemaNonClass<
 		`com.fluidframework.leaf.${Name}`,
 		NodeKind.Leaf,
-		T,
-		T,
-		true
+		T, // TNode
+		T, // TInsertable
+		true // ImplicitlyConstructable
```

Comment:
I don't know how the doc generation treats these. The don't come out well in the report api.md files. The formatting makes it look like the comment applies to the thing after.
I think 
```suggestion
		/* TNode */ T,
		/* TInsertable */ T,
		/* ImplicitlyConstructable */ true
```
is a good pattern for documenting parameters.

Reply:
I'm pretty sure the issue is just a bug in API-documenter's comment stripping, but your proposal works around the issue, and works in more cases, so I've updated to it.